x-pack/plugins/endpoint/public/applications/endpoint/index.tsx

```diff
@@ -33,6 +33,7 @@ export function renderApp(coreStart: CoreStart, { appBasePath, element }: AppMou
 interface RouterProps {
   basename: string;
   store: Store;
+  coreStart: CoreStart;
```

Comment:
@kqualters-elastic - maybe not for this PR, but we should consider using the [kibana-react](https://github.com/elastic/kibana/blob/master/src/plugins/kibana_react/README.md) components for sharing `coreStart` in the app. I think the use of hooks rather than to force `coreStart` down to each route's View would be better DX. If we did not want to use the kibana-react components, we could just create our own react `Context` and set of hooks.

Reply:
agreed, updated to do just that.